### DISCUSSION TITLE
Social Cards : Text Overflow: shrink by default

### DISCUSSION THREAD

**Discussion by KeithIMyers** [squidfunk/mkdocs-material#6058] [Q&A] [answered]
Good Morning All,
Is there a easy way to set the Text Overflow on social cards to "shrink" by default or can this only be done with a custom card theme?

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. You currently have to copy one of our layouts for this, but it's definitely a reasonable request to add a new option to our [default layouts](https://squidfunk.github.io/mkdocs-material/plugins/social/#layouts), i.e., `overflow`. If you would like to see this in one of the next releases, please open a [change request](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/).
